nototools/unicode_data.py

```diff
@@ -784,7 +784,7 @@ def _read_emoji_data(lines):
       continue
 
     # discourage lots of redundant copies of seq_type
```

Comment:
delete comment, no longer applies?

Reply:
Yes, no longer applies when  the intern() is gone!